### PR TITLE
Activecoupons

### LINKED ISSUE #49: Active Coupon- Admin Page

### PR DESCRIPTION
**Pull request by bertacsevercan**
#49 
- Add active coupons component into Admin Dashboard.
- In this component, the user can see the total active coupons and when one of them used, they can mark it as used, so, later on, it will show up in the Used Coupons component.
- When one coupon marked used, the total amount will be decreased. 
- The button is disabled for the total less than 1.
- Desktop
Ekran Resmi 2020-11-19 17 25 59

- İpad pro
Ekran Resmi 2020-11-19 17 26 04

- iPad 
Ekran Resmi 2020-11-19 17 29 22

- Mobile
Ekran Resmi 2020-11-19 17 26 42

### CODE REVIEW

File: src/components/ActiveCoupons/index.jsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        totalSoldCoupons["100₺"].push(soldCoupons[i].docId);
+        break;
+      case "150₺":
+        totalSoldCoupons["150₺"].push(soldCoupons[i].docId);
+        break;
+      case "200₺":
+        totalSoldCoupons["200₺"].push(soldCoupons[i].docId);
+        break;
+      default:
+        console.log("err");
```
Comment (halitbatur) [inline src/components/ActiveCoupons/index.jsx]: lets use an alert here and remove the console.log

File: src/components/ActiveCoupons/index.jsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                </Text>
+                <br />
+                <br />
+                {isTesting ? (
+                  <button type="button">{t("activeCoupons.button")}</button>
+                ) : (
+                  <Button
+                    disabled={!(card.text.length > 0)}
+                    onClick={() =>
+                      markUsed(card.text[card.text.length - 1], card.title)
```
Comment (halitbatur) [inline src/components/ActiveCoupons/index.jsx]: I know its a mock data, but why are we using the card.text as an id ?

**Review by halitbatur — APPROVED** [review]

File: src/components/ActiveCoupons/index.jsx (hunk 2 above)
Comment (bertacsevercan) [reply src/components/ActiveCoupons/index.jsx]: card text has the docids inside, it's actually an array. card.text contains the unique ids for each coupon

Comment (halitbatur) [reply src/components/ActiveCoupons/index.jsx]: @bertacsevercan can we change the name so it's more clear ?

**Comment by halitbatur** [comment]
@bertacsevercan merge it with the update version of the Dev

**Review by halitbatur — APPROVED** [review]

**Review by ihab-sensei — APPROVED** [review]
LGTM!

**Comment by bertacsevercan** [comment]
@halitbatur Resolved conflicts!

**Pull request merged**